pkg/uefi/meregion_test.go

```diff
@@ -91,3 +92,26 @@ func TestFindFPTSignature(t *testing.T) {
 		})
 	}
 }
+
+func TestParseFPT(t *testing.T) {
+	t.Run("parse ME FPT", func(t *testing.T) {
+		fixtureFile := "../../data/PurleySiliconBinPkg/MeFirmware/IgnitionFirmware/MeRegion.bin"
+		meRegion, err := os.ReadFile(fixtureFile)
+		if err != nil {
+			t.Fatalf("could not read test fixture %q", fixtureFile)
+		}
+		expectedEntries := 10
+
+		fpt, err := NewMEFPT(meRegion)
+		if err != nil {
+			t.Errorf("expected no error reading a valid ME FPT, got %v", err)
```

Comment:
The general form of these errors messages, is the got, want format, e..g
reading ME FPT: got %v, want nil

Reply:
Yea I saw that in the earlier suggestions and find it... irritating/lossy.
Why not provide the detailed idea in the test - which is the spec of behavior?
What I'm saying here is: "I expect this function to return either a result or an error, but never both".
Anyway, will change it if that's the way to go.